graylog2-web-interface/src/components/widgets/WidgetFooter.jsx

```diff
@@ -47,25 +47,16 @@ const WidgetFooter = React.createClass({
     }
 
     // if we have a tooltip, we disable the button link and instead show a tooltip on hover
-    const title = this.props.replayToolTip ? null : 'Replay search';
-    const href = this.props.replayToolTip ? null : this.props.replayHref;
-    let replay = (
-      <Button bsStyle="link" className="btn-text" title={title} href={href}>
-        <i className="fa fa-play" />
-      </Button>
+    const replay = this.props.replayDisabled ? <div /> : (
```

Comment:
Is there a reason why you have chosen to render an empty `<div>` instead of `null`? We do the latter in most of the code base when we want to render nothing but need to assign a value to a variable. If there is a reason why `null` is a bad idea in this case, we should still use a `<span>` instead of a `<div>`, as the former has no semantic implications.